### DISCUSSION TITLE
for what lhc-php-resque is needed?

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1855] [General]
can someone explain what the extension lhc-php-resque is needed for? is it important to have or not? i installed it like it is described here https://github.com/LiveHelperChat/lhc-php-resque. 

like in step 6. i executed `REDIS_BACKEND=localhost:6379 INTERVAL=5 REDIS_BACKEND_DB=1 VERBOSE=1 COUNT=1 QUEUE='*' /usr/bin/php resque.php` and when starting a chat i could see some debug notice in command prompt about new chat but i can't understand why i should do that at all with this extension. can someone please explain the use of this? 

in setp 7. there is also `Create folder named cron in root Live Helper Chat folder` but why should you create this **_cron_** folder in lhc root folder? normally i would think that the cronjob are configured with `crontab `command and there you don't need a **_cron_** folder

in step 8. there is an example for a schedule command. why should you need this?

**Comment by remdex** [comment]
It sends and does webhooks events in the background. https://github.com/LiveHelperChat/lhc-php-resque so UI is not blocked and various other extensions use it.

**Reply by comsyspro** [reply]
so do you suggest to use it? does it have any positive effect when i install it like in your description and run the shell scripts via crontab in step 7.?
```
@reboot cd /var/www/web/extension/lhcphpresque/doc/ && ./phpresque.sh >> /dev/null 2>&1
40 7 * * * /bin/touch /var/www/web/extension/lhcphpresque/doc/runresque.lock > /dev/null 2>&1
* * * * * cd /var/www/web/extension/lhcphpresque/doc && ./resque.sh >> /dev/null 2>&1
```
or do i need to do anything else with lhc-php-resque after crontab is setup?
